swan-lake/learn-the-platform/run-in-the-cloud/function-as-a-service/azure-functions.md

```diff
@@ -214,4 +219,175 @@ Once the function is deployed, add an item to the collection.
+   4.30MB ballerina.http/2.types                              13.12MB byte[] for embedded resources
+   3.02MB ballerina.io/1                                       6.70MB java.lang.Class
+   1.88MB ballerina.file/1                                     5.11MB byte[] for java.lang.String
+   1.75MB ballerina.jwt/2                                      4.64MB java.lang.String
+   1.60MB sun.security.ssl                                     3.63MB byte[] for general heap data
+   1.33MB ballerina.oauth2/2                                   2.31MB com.oracle.svm.core.hub.DynamicHubCompanion
+   1.25MB java.lang.invoke                                     1.29MB byte[] for reflection metadata
+   1.22MB com.sun.media.sound                                983.89KB java.lang.String[]
+   1.06MB ballerina.lang$0046query/0                         934.65KB c.o.svm.core.hub.DynamicHub$ReflectionMetadata
+  57.84MB for 914 more packages                                6.28MB for 3340 more object types
+------------------------------------------------------------------------------------------------------------------------
+                        26.1s (7.6% of total time) in 71 GCs | Peak RSS: 7.77GB | CPU load: 6.64
+------------------------------------------------------------------------------------------------------------------------
+Produced artifacts:
+ <package_dir>/azure_functions_native/target/azure_functions/azure_functions_native (executable)
+ <package_dir>/azure_functions_native/target/azure_functions/azure_functions_native.build_artifacts.txt (txt)
+========================================================================================================================
+Finished generating 'azure_functions_native' in 5m 44s.
+
+        @azure_functions:Function: get-hello
+
+        Execute the command below to deploy the function locally:
+        $ bal build --native --cloud="azure_functions_local"
+        $ func start --script-root target/azure_functions
+
+        Execute the command below to deploy Ballerina Azure Functions:
+        $ bal build --native --cloud="azure_functions"
+        $ func azure functionapp publish <function_app_name> --script-root target/azure_functions
+```
+
+Execute the following command to run the azure function locally.
+```bash
+$ func start --script-root target/azure_functions
+
+Azure Functions Core Tools
+Core Tools Version:       4.0.4895 Commit hash: N/A  (64-bit)
+Function Runtime Version: 4.13.0.19486
+
+[2023-02-14T11:14:19.058Z] Worker process started and initialized.
+
+Functions:
+
+        get-hello: [GET] http://localhost:7071/hello
+```
+
+Execute the following command to test the function locally.
+
+```bash
+$ curl "http://localhost:7071/hello?name=Jack"
+Hello, Jack!
+```
+
+### Deploy the function
+
+Now you can build the package for azure functions cloud. This will perform the compilation inside docker image to make it compatible with azure functions cloud environment.
```

Comment:
```suggestion
Now, you can build the package for the Azure Functions cloud. This will perform the compilation inside the Docker image to make it compatible with the Azure Functions cloud environment.
```